### PR TITLE
useMutableState hook for React via createMutable

### PR DESCRIPTION
**Pull request by steve8708**

### CODE REVIEW

**Comment by ryansolid** [comment]
Hmm.. only thing is so far I've kept the naming congruent with Solid's naming.. Ie.. if `createState` -> `useState`.  Now it's arguable that I probably should have been more explicit with `createMutable` but I think I want to keep the consistent naming pattern.

**Comment by steve8708** [comment]
ah yes makes sense, updated 👍

File: packages/react-solid-state/src/index.js [hunk 1]
```diff
@@ -23,7 +24,7 @@ let inSolidEffect = false;
 function trackNesting(args) {
   const fn = args[0];
   return [
-    function() {
+    function () {
```
Comment (steve8708) [inline packages/react-solid-state/src/index.js:27]: just realized this got added from prettier in my IDE - can revert if not desired

Comment (ryansolid) [reply packages/react-solid-state/src/index.js:27]: Don't worry about it.

**Pull request merged**